layouts/members/single.html

```diff
@@ -321,52 +321,87 @@ <h6 class="pb-2 mb-5 border-bottom text-uppercase">
                                 </div>
                                 <h5 class="mb-1">Jack Aboutboul</h5>
                                 
                                 <div class="small text-uppercase mb-1">AlmaLinux OS Community Manager</div>
-                                <div class="small fw-light">CloudLinux Inc</div>
+                                <div class="small fw-light">Microsoft, INC</div>
                             </div>
                         </div>
                     
                         <div class="col-xl-3 col-sm-6 mb-5">
                             <div class="py-5 px-4 text-center al-member-item">
                                 <div>
-                                    <img class="img-fluid rounded-circle mb-3 img-thumbnail" src="/brands/jesse.jpg" alt="Jesse Asklund" style="max-width: 180px;">
+                                    <img class="img-fluid rounded-circle mb-3 img-thumbnail" src="/board/jesse.jpg" alt="Jesse Asklund" style="max-width: 180px;">
                                 </div>
                                 <h5 class="mb-1">Jesse Asklund</h5>
                                 
                                 <div class="small text-uppercase mb-1">{{ i18n "Global head of customer experience for" }} WebPros</div>
-                                <div class="small fw-light">cPanel</div>
+                                <div class="small fw-light">WebPros</div>
                             </div>
                         </div>
                     
                         <div class="col-xl-3 col-sm-6 mb-5">
                             <div class="py-5 px-4 text-center al-member-item">
                                 <div>
-                                    <img class="img-fluid rounded-circle mb-3 img-thumbnail" src="/brands/simon.jpg" alt="Simon Phipps" style="max-width: 180px;">
+                                    <img class="img-fluid rounded-circle mb-3 img-thumbnail" src="/board/simon.jpg" alt="Simon Phipps" style="max-width: 180px;">
                                 </div>
                                 <h5 class="mb-1">Simon Phipps</h5>
-                                
+								
                                     <div class="small text-uppercase mb-1">{{ i18n "open source advocate and a former president of the" }} Open Source Initiative</div>
-                                
                                 <div class="small text-uppercase mb-1"></div>
                                 <div class="small fw-light"></div>
                             </div>
                         </div>
+						
+     				 <div class="col-xl-3 col-sm-6 mb-5">
+                        <div class="py-5 px-4 text-center al-member-item">
+                            <div>
+                                <img class="img-fluid rounded-circle mb-3 img-thumbnail" src="/board/default.png" alt="AlmaLinux OS Logo" style="max-width: 180px;">
```

Comment:
Let's get headshots from Dan and Moshe soon.  I always hate placeholder images :)

Reply:
I'm sure we will! I've asked for them, and when they appear I'll update the site. :)